protected/humhub/libs/Helpers.php

```diff
@@ -187,6 +188,94 @@ public static function GetUniqeId()
         return str_replace('.', '', uniqid('', true));
     }
 
+    /**
+     * @param mixed $value value to be tested or converted
+     * @param bool $strict indicates if strict comparison should be performed:
+     * ``
+     * - if True, `$value` must already be of type `int`.
+     * - if False, a conversion to `int` is attempted.
+     * ``
+     *
+     * @since 1.16
+     */
+    public static function checkBool($value, bool $strict = false): ?bool
```

Comment:
@martin-rueegg Why do we need the methods `checkBool` and `checkFloat` if they are not used?